messagebus/src/vespa/messagebus/testlib/custompolicy.cpp

```diff
@@ -38,10 +37,8 @@ CustomPolicy::select(RoutingContext &context)
     str.append(" }.");
     context.trace(1, str);
     context.setSelectOnRetry(_selectOnRetry);
-    for (std::vector<uint32_t>::iterator it = _consumableErrors.begin();
-         it != _consumableErrors.end(); ++it)
-    {
-        context.addConsumableError(*it);
+    for (unsigned int & _consumableError : _consumableErrors) {
```

Comment:
Would it be better to leave this as `for (uint32_t ...` to ensure same type as the underlying vector (or just `auto` it)? Also consider removing ref to primitive and leading underscore on temporary name

Reply:
clion automatic fix takes a few short cuts...